layouts/community/ortho_5x12/xyverz/config.h

```diff
@@ -7,16 +7,19 @@
 
 /* ws2812 RGB LED */
 #undef RGB_DI_PIN
-#define RGB_DI_PIN B3
-#undef RGBLED_NUM
-#define RGBLIGHT_ANIMATIONS
-#define RGBLED_NUM 11     // Number of LEDs
-#undef RGBLIGHT_HUE_STEP
-#define RGBLIGHT_HUE_STEP 8
-#undef RGBLIGHT_SAT_STEP
-#define RGBLIGHT_SAT_STEP 8
-#undef RGBLIGHT_VAL_STEP
-#define RGBLIGHT_VAL_STEP 8
+#if defined(KEYBOARD_preonic)
```

Comment:
This won't work, unfortunately.  It has to be the full keyboard, with revision listed. 

Optionally, adding this to your rules.mk may work: 
```makefile
ifneq (,$(findstring preonic,$(KEYBOARD)))
  OPT_DEFS += -DKEYBOARD_preonic
endif
```
This will define `KEYBOARD_preonic` for all of the preonic boards.

Reply:
Specified revision. I have an Rev2.